### PR TITLE
Mention the corresponding Babel plugins

### PR DESCRIPTION
**Pull request by dandv**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -111,6 +111,11 @@ class User {
 * Node 8.10+
 * mongoose ^5.9.2
 * `emitDecoratorMetadata` and `experimentalDecorators` must be enabled in `tsconfig.json`
+  * if you use Babel (e.g. with Jest), it will need the corresponding plugins to complement the [typescript preset](https://babeljs.io/docs/en/babel-preset-typescript):
+  
+        ['@babel/plugin-proposal-decorators', { legacy: true }],
```
Comment (hasezoey) [inline README.md:116]: please wrap this in an code block instead of indentation and explain what this is for

File: README.md [hunk 2]
```diff
@@ -111,6 +111,11 @@ class User {
 * Node 8.10+
 * mongoose ^5.9.2
 * `emitDecoratorMetadata` and `experimentalDecorators` must be enabled in `tsconfig.json`
+  * if you use Babel (e.g. with Jest), it will need the corresponding plugins to complement the [typescript preset](https://babeljs.io/docs/en/babel-preset-typescript):
```
Comment (hasezoey) [inline README.md:114]: please fix the extra 2 space indentation

Comment (dandv) [reply README.md:114]: If I removed those, the block didn't get formatted as code by GitHub. I'll see about making it a code block.

**Comment by dandv** [comment]
Using Babel is a bad idea. I've [switched to ts-jest](https://github.com/dandv/typegoose-jest-enums/commit/c41f688eb3372c8272a2d91677762832d369ef90) and avoided other problems. This PR should be undone, and `ts-jest` should be recommended.

**Comment by hasezoey** [comment]
yes using babel is an bad idea, but for those who still want to babel with typegoose, it should at least mention the required plugins

**Comment by dandv** [comment]
Oh, I've just done that, didn't see your update

**Comment by hasezoey** [comment]
@dandv it seems like the babel documentation has become outdated, could you maybe update if you know how to use it?

**Comment by dandv** [comment]
Sorry, I haven't worked on this in a very long time.

**Pull request merged**